src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -267,22 +281,17 @@ class ServerListWindow final : public Window
         if (text.empty())
             return;
 
-        std::string temp = static_cast<std::string>(text);
+        auto temp = static_cast<u8string>(text);
```

Comment:
This looks a bit dubious, text is `std::string_view`, it should be probably `auto temp = u8string{text};`

Reply:
Yea, good call. It's been a learning experience this PR with not just updating things but evaluating what they are and the better way to implement them: [c9b3e8f](https://github.com/OpenRCT2/OpenRCT2/pull/20004/commits/c9b3e8f185d96dacacb721875134e639864f3ea4)